ui/raidboss/timeline.js

```diff
@@ -199,45 +199,44 @@ export class Timeline {
         continue;
       const originalLine = line;
 
-      let match = regexes.ignore.exec(line);
-      if (match) {
-        this.ignores[match[1]] = true;
+      const [, ignoreId] = regexes.ignore.exec(line) || [];
+      if (ignoreId) {
+        this.ignores[ignoreId] = true;
         continue;
       }
 
-      match = regexes.tts.exec(line);
-      if (match) {
+      const [, ttsId, ttsSeconds, ttsCommand, ttsText] = regexes.tts.exec(line) || [];
```

Comment:
Do you think it's worth rewriting the regexes as well to have named capture groups rather than using ids by position? e.g. changing the above to `tts: /^alertall\s+"(?<ttsId>[^"]*)"\s+before\s+(?<ttsSeconds>-?[0-9]+(?:\.[0-9]+)?)\s+(?<ttsCommand>sound|speak\s+"[^"]*")\s+"(?<ttsText>[^"]*)"$/,`, and then destructring into an object instead of an array?